utilities.py

```diff
@@ -449,4 +455,15 @@ def def_chunk_size(data_shape):
         if durantion < chunk_size:
             chunk_size = durantion
 
-        return  chunk_size
\ No newline at end of file
+        return  chunk_size
+
+
+def clean_path(path):
```

Comment:
I created this function to help to clean the path so I don't have to do the ifs every time